File: src/main/java/com/tibiawiki/domain/enums/Star.java
package com.tibiawiki.domain.enums;

import com.fasterxml.jackson.annotation.JsonValue;

public enum Star {

    ONE(1),
    TWO(2),
    THREE(3),
    FOUR(4),
    FIVE(5);

    private int number;

    Star(int number) {
        this.number = number;
    }

    @JsonValue
    public int getNumber() {
        return number;
    }
}

File: src/main/kotlin/com/tibiawiki/domain/enums/Star.kt
package com.tibiawiki.domain.enums

import com.fasterxml.jackson.annotation.JsonValue

enum class Star(
    @get:JsonValue val number: Int
) {
    ONE(1),
    TWO(2),
    THREE(3),
    FOUR(4),
    FIVE(5)
}
